cmd/registry-replacer/main.go

```diff
@@ -200,9 +209,8 @@ func replacer(
 	ensureCorrectPromotionDockerfile bool,
 	ensureCorrectPromotionDockerfileIgnoredrepos sets.String,
 	promotionTargetToDockerfileMapping map[string]dockerfileLocation,
-	majorMinor ocpbuilddata.MajorMinor,
-	credentials *usernameToken,
-) func(*api.ReleaseBuildConfiguration, *config.Info) error {
+	majorMinor ocpbuilddata.MajorMinor, credentials *usernameToken,
+	configResolver func(config api.ReleaseBuildConfiguration) (api.ReleaseBuildConfiguration, error)) func(*api.ReleaseBuildConfiguration, *config.Info) error {
```

Comment:
Could you keep the new line before the return signature, this is pretty confusing to read :grimacing: